app/models/duplicate_checker_job.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class DuplicateCheckerJob < ApplicationRecord
```

Comment:
This should probably be called `DuplicateCheckerJobRuns` because you're not serializing the actual **job** itself. You are serializing the information that the job ran (and when) and what its results are

Reply:
Don't forget to rename the table in the migration and schema.rb accordingly

Reply:
Done